### ISSUE TITLE
Changing values on a non complete workflow is impossible

### ISSUE THREAD

**Issue by Mikerhinos** [googincheng/ComfyUX#7]
I love this tool :)

Though I have a long workflow in different stages, where I mute some nodes to "lock" it from going further (detailers, upscaling, etc. until I get what I want step by step).
It seems that when I generate an image on a part of workflow, it's then impossible to change a value in a ComfyUX favourited field as it will get back the value from the one in the ComfyUI node.

**Comment by googincheng** [comment]
Thks for your like and feedback, I'll check it out

**Comment by googincheng** [comment]
I can't re produce the case you mentioned, can you share a video or more info? node name?

**Comment by Mikerhinos** [comment]
Yes it seems it's not related to the disabled node, and after an hour of use I don't have the problem anymore today, maybe I needed to reboot the pc to finish the installation of some library or something.
If I have the problem again I'll record it in video, if not sorry for that :)

**Comment by googincheng** [comment]
It's fine, appreciated for that

**Comment by Mikerhinos** [comment]
https://github.com/user-attachments/assets/a99e4b7d-5761-4f4b-add1-08d59ff44b42

Happened again, I'm using Pinokio, no error in console or browser :(

**Comment by googincheng** [comment]
Thks for the video, I'am working on it

**Comment by googincheng** [comment]
Can you update to the latest version to test, I still can't catch the bug.
